blogs/converter.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const marked = require('marked');
+
+var listOfDirectoryFiles = fs.readdirSync('markdown-blogs');
```

Comment:
I think we should update the directories to the pattern
```
/blog
--html-blog.html
--/markdown
--/markdown/file.md
```
so markdown are inside a folder of blogs then the html files are at same level as that directory

Reply:
fixed.